lib/modules/apostrophe-templates/index.js

```diff
@@ -696,5 +696,30 @@ module.exports = {
       req.data.aposBodyClasses = (req.data.aposBodyClasses ? (req.data.aposBodyClasses + ' ') : '') +
         bodyClass;
     };
+
+    // Add a body attribute to be emitted when the page is rendered. This information
+    // is attached to `req.data`, where `req.data.aposBodyDataAttributes` is built up
+    // using `name` as the attribute name which is automatically prepended with "data-"
+    // and the optional `value` argument
+    // Also receives a single object or an array of objects with name and value key pairs
```

Comment:
No longer quite accurate, we removed the array syntax but you're still talking about it in the doc comment.